benchmarks/src/lib.rs

```diff
@@ -16,27 +16,10 @@
 // under the License.
 
 //! DataFusion benchmark runner
-use datafusion::error::Result;
-use structopt::StructOpt;
-
-pub mod run;
+pub mod clickbench;
 pub mod tpch;
 
+mod options;
+mod run;
+pub use options::CommonOpt;
 pub use run::{BenchQuery, BenchmarkRun};
-
```

Comment:
this was dead code I accidentally introduced in https://github.com/apache/arrow-datafusion/pull/7054

The actual entrypoint for the dfbench binary is in https://github.com/apache/arrow-datafusion/blob/main/benchmarks/src/bin/dfbench.rs